projects/packages/a8c-mc-stats/composer.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "automattic/jetpack-a8c-mc-stats",
 	"description": "Used to record internal usage stats for Automattic. Not visible to site owners.",
-	"type": "library",
+	"type": "jetpack-library",
```

Comment:
Do we want to change every package, or only the ones that have i18n? I don't have much of an opinion either way. Changing all means less problem if i18n is added later, but also makes it less clear which actually need the special treatment.

The ones that currently have calls to i18n functions seem to be assets, backup, config, connection, connection-ui, heartbeat, identity-crisis, jitm, lazy-images, licensing, my-jetpack, partner, password-checker, post-list, search, sync, and tracking.

Reply:
I chose to move all of the packages because I thought it would decrease the chances of problems if i18n is added to packages later. I don't have a strong opinion on this either.

@zinigor - do you have a preference?

Reply:
I literally just asked Kim that in Slack, having this page open before my eyes 🤦🏽 . I had the same question, but I do agree that preventing accidents in the future is a good reason. Plus, having all library packages in a separate vendor folder feels right, if that makes sense. Let's leave it like that.